modules/core/src/test/java/org/apache/fluo/core/data/BytesBuilderTest.java

```diff
@@ -71,6 +71,17 @@ public void testSingleByte() {
   }
 
   @Test
+  public void testArraySection() {
+    BytesBuilder bb = Bytes.newBuilder();
+
+    bb.append(new byte[] {'c', (byte) 0, (byte) 127, (byte) 128, (byte) 255}, 0, 3);
+
+    Bytes bytes = bb.toBytes();
+    Assert.assertEquals(Bytes.of(new byte[] {'c', (byte) 0, (byte) 127, (byte) 128, (byte) 255}),
```

Comment:
Could call the `Bytes.of(byte[],int offset, int len)` method to produce the expected data.